app/src/features/Joins/JoinSelects.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import { IconButton, Grid, Typography } from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+
+import ColumnSelects from "features/Columns/ColumnSelects";
+import { PendingJoin } from "features/Mappings/mappingSlice";
+import { Owner } from "services/api/generated/api.generated";
+
+type JoinSelectsProps = {
+  join: PendingJoin;
+  owner?: Owner;
+  disableDelete?: boolean;
+  onChange?: (join: PendingJoin) => void;
+  onDelete?: (joinId: string) => void;
+};
+
+const JoinSelects = ({
+  join,
+  owner,
+  onChange,
+  onDelete,
+  disableDelete,
+}: JoinSelectsProps): JSX.Element => {
+  const [leftColumn, rightColumn] = join.columns;
+
+  const handlePKTableAChange = (table?: string) => {
```

Comment:
why PrimaryKey ?

Reply:
It's just a wrong naming. Should have called it `table` & `column`